src/java/fr/paris/lutece/portal/web/dashboard/DashboardJspBean.java

```diff
@@ -75,7 +74,7 @@
 
     // JSP
     private static final String ANCHOR_ADMIN_DASHBOARDS = "adminHomePageManagement";
-    private DashboardService _service = DashboardService.getInstance( );
+    private transient DashboardService _service = DashboardService.getInstance( );

```

Comment:
As before, this does not make the class properly serializable.

Reply:
Will be corrected